uwsgi.ini

```diff
@@ -18,6 +18,7 @@ offload-threads = 2
 thunder-lock =
 if-env = DEV_ENV
 python-autoreload = 1
+harakiri = 30
```

Comment:
Might be worth checking with Nathan to see if this change is good for production.